test/fixtures/data.js

```diff
@@ -59,6 +59,7 @@ const firstTimerPrivatePR = readJSON('first_timer_pr_with_private_email.json');
 const semverMajorPR = readJSON('semver_major_pr.json');
 const fixAndRefPR = readJSON('pr_with_fixes_and_refs.json');
 const conflictingPR = readJSON('conflicting_pr.json');
+const incorrectConfigPR = readJSON('incorrect_config_pr.json');
```

Comment:
Maybe renaming this to something like `emptyProfilePR`

Reply:
Ah, yeah makes sense since this seems to be something other than incorrect config i was thinking.